test/docker/config-template.toml

```diff
@@ -1,2 +1,10 @@
+vote_extensions_enable_height = 0
+check_tx_delay = "1000s"
+finalize_block_delay = "1000ms"
+prepare_proposal_delay = "1000ms"
+process_proposal_delay = "1000ms"
+vote_extension_delay = "20ms"
+vote_extension_size = 8192
+
```

Comment:
[not 100% sure] These are parameters for the e2e app, but this file is used to build CometBFT Docker images without the e2e app.
Could you double-check where this file is used? (if the e2e app is not involved, please revert this hunk)